addon/helpers/is-clipboard-supported.js

```diff
@@ -1,5 +1,4 @@
 import Ember from 'ember';
-/* global Clipboard */
 
-export const isClipboardSupported = Clipboard.isSupported;
+export const isClipboardSupported = window.Clipboard ? window.Clipboard.isSupported: () => false;
```

Comment:
Can we make this the following:

```js
window && window.Clipboard ? window.Clipboard.isSupported : () => false;
```
The docs suggest testing for the presence of `window`
https://ember-fastboot.com/docs/addon-author-guide#how-to-fix-it